content/code_samples/escrow/websocket/account_objects-response-sender.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
account_objects-response-sender.json has been deleted. It is replaced by account_objects_sender.json.